bazel/ray_deps_setup.bzl

```diff
@@ -212,12 +212,14 @@ def ray_deps_setup():
 
     # OpenCensus depends on Abseil so we have to explicitly pull it in.
     # This is how diamond dependencies are prevented.
+    #
+    # TODO(owner): Upgrade abseil to latest version after protobuf updated, which requires to upgrade `rules_cc` first.
```

Comment:
Error when I was trying to further bump up version:
```
external/com_google_protobuf/src/google/protobuf/arena.cc:472:49: error: 'Layout' in namespace 'absl::lts_20240116::container_internal' does not name a template type
  472 |   using layout_type = absl::container_internal::Layout<
      |                                                 ^~~~~~
external/com_google_protobuf/src/google/protobuf/arena.cc:494:20: error: 'layout_type' does not name a type
  494 |   constexpr static layout_type Layout(size_t n) {
      |                    ^~~~~~~~~~~
```
which means our protobuf and abseil are too old so google side directly removes the interface :(